front/src/reducers/osrdconf/operationalStudiesConf/simulationConfReducers.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, expect } from 'vitest';
+import { createStoreWithoutMiddleware } from 'Store';
+
+import { infraState } from 'reducers/infra';
+import { defaultCommonConf } from 'reducers/osrdconf/osrdConfCommon';
+import testCommonConfReducers from 'reducers/osrdconf/osrdConfCommon/__tests__/utils';
+import { operationalStudiesConfSlice } from 'reducers/osrdconf/operationalStudiesConf';
+
+const createStore = () =>
+  createStoreWithoutMiddleware({
+    [operationalStudiesConfSlice.name]: {
+      ...defaultCommonConf,
+      ...infraState,
```

Comment:
infraState is already in defaultCommonConf
```suggestion
```

Reply:
Done